services/checks/checks.go

```diff
@@ -416,6 +417,111 @@ func (s *Service) EnableChecks(checkNames []string) error {
 	return nil
 }
 
+// ChangeInterval changes a check's interval to the value received from the UI.
+func (s *Service) ChangeInterval(checkName string, interval managementpb.SecurityCheckInterval) error {
+	checks := s.GetAllChecks()
+	if len(checks) == 0 {
+		return errors.New("no checks loaded")
+	}
+
+	for _, c := range checks {
+		if c.Name == checkName {
+			s.l.Error(c.Name)
```

Comment:
why log this?

Reply:
no need, I guess I forgot to remove it after debuging thanks